cfme/common/vm.py

```diff
@@ -522,20 +521,19 @@ class VM(BaseVM):
     TO_RETIRE = None
 
     retire_form = Form(fields=[
-        ('date_retire', {
-            version.LOWEST: date_retire_element,
-            "5.5": AngularCalendarInput(
-                "retirement_date", "//label[contains(normalize-space(.), 'Retirement Date')]")}),
+        ('date_retire', AngularCalendarInput(
+            "retirement_date", "//label[contains(normalize-space(.), 'Retirement')]")),
```

Comment:
**REQUIRED:** Specify locator
`"//label[contains(normalize-space(.), 'Retirement')]"` has two possible values in page, something can be added in future with  'Retirement' in locator